package.json

```diff
@@ -51,10 +51,13 @@
     "i18next-browser-languagedetector": "2.1.0",
     "i18next-xhr-backend": "1.5.0",
     "immutable": "3.8.2",
+    "lodash": "4.17.15",
+    "lodash.throttle": "4.1.1",
```

Comment:
you can remove this line "lodash.throttle

Reply:
Thanks. Removed.